portfolio/src/main/webapp/chart-script.js

```diff
@@ -20,6 +23,33 @@ function init() {
       });
 }
 
+/**
+ * Initializes country selector
+ */
+function initSelector() {
+  const countrySelector = document.getElementById('country-selector');
+
+  const checkboxes = document.createElement('ul');
+  for (let i = 0; i < checkboxValues.length; i++) {
```

Comment:
`for (const value of checkboxValues)` is cleaner if you don't need to refer to the index

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of